bitbots_quintic_walk/src/walk_node.cpp

```diff
@@ -492,22 +482,7 @@ void WalkNode::publishOdometry(WalkResponse response) {
   twist.angular.z = current_request_.angular_z * walk_engine_.getFreq() * 2;
 
   odom_msg_.twist.twist = twist;
-  pub_odometry_.publish(odom_msg_);
-
-  if (publish_odom_tf_) {
```

Comment:
You removed the usage `publish_odom_tf_`, you should also remove it from the config files and the header file.